### PR TITLE
Prepublication edits for IUA Rev. 2.3

### PR DESCRIPTION
**Pull request by MaryLJ**
Closes # 

## 📑 Description

updated date, updated Appendix A and B titles and applicable TOC entries, capitalization updates, updated Appendix A and B, updated Section 9, added Section 10 and added to TOC, punctuation updates, updated link in Section 34.1, updated some heading level formats (bold to applicable # indicating level), spell check

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed
- [x] I have selected a committee co-chair to review the PR

### CODE REVIEW

File: IHE_ITI_Suppl_IUA.md [hunk 1]
```diff
@@ -813,13 +815,13 @@ If the request is valid and all access policy criteria are met, the Authorizatio
 
 If the authorization request is invalid, the Authorization Server shall react as defined in [OAuth 2.1, Section 4.1.2.1].
 
-###### 3.71.4.1.3.1 Client Credential grant type
+##### 3.71.4.1.3.1 Client Credential grant type
```
Comment (JohnMoehrke) [inline IHE_ITI_Suppl_IUA.md:818]: why did you change the header level to 5? The numbers indicate there are 6.

Comment (JohnMoehrke) [reply IHE_ITI_Suppl_IUA.md:818]: is this to get around a limitation in markdown? I see other deep header changes too.

**Review by JohnMoehrke — COMMENTED** [review]
question on markdown header levels

**Comment by MaryLJ** [comment]
I thought markdown only supported level 5

 

Mary Jungers | Documentation Specialist 

IHE International | IHE.net

**Comment by JohnMoehrke** [comment]
I checked.. github markdown supports 6. So we could add back in support for 6. I think this was the original. 
Note when we needed 7, that is why we just used bold text, so that we didn't have a messed up table of content.

**Comment by JohnMoehrke** [comment]
I reverted these. I also confirmed that this will produce the right html.

**Review by JohnMoehrke — APPROVED** [review]
looks good with header 6
